recipes/spirv-tools/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+cmake_minimum_required(VERSION 2.8.12)
+project(test_package)
+
+if( ${BUILD_C_TEST_PACKAGE} )
+    set(CMAKE_C_STANDARD 11)
+else()
+    set(CMAKE_CXX_STANDARD 11)
+endif()
+
+set(CMAKE_VERBOSE_MAKEFILE TRUE)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+if( ${BUILD_C_TEST_PACKAGE} )
```

Comment:
build both, its fine. Just add something like `set_property(TARGET test_package_c PROPERTY LINKER_LANGUAGE CXX)`

here is what I do usually in this case:
```cmake
cmake_minimum_required(VERSION 2.8.11)
project(test_package)

include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
conan_basic_setup()

add_executable(test_package_c test_package.c)
target_link_libraries(test_package_c ${CONAN_LIBS})
set_property(TARGET test_package_c PROPERTY LINKER_LANGUAGE CXX)

add_executable(test_package_cpp test_package.cpp)
target_link_libraries(test_package_cpp ${CONAN_LIBS})
set_property(TARGET test_package_cpp PROPERTY CXX_STANDARD 11)
```